include/deal.II/sundials/n_vector.templates.h

```diff
@@ -203,10 +284,39 @@ namespace SUNDIALS
                  N_Vector           y,
                  N_Vector           z);
 
+      template <
+        typename VectorType,
+        std::enable_if_t<is_dealii_compatible_distributed_vector<VectorType>,
+                         int> = 0>
```

Comment:
Above you use the idiom that the dummy argument is of pointer type, here it's an `int`. I don't think it matters much which you choose, but it would be nice to be consistent :-) I would go with the `nullptr` thing because then you don't have to provide the second template argument to `std::enable_if_t`.

Same for the places below of course.